examples/lang-err-integration-tests/call-builder/lib.rs

```diff
@@ -55,14 +55,42 @@ mod call_builder {
                 }
             }
         }
+
+        #[ink(message)]
+        pub fn call_instantiate(
+            &mut self,
+            code_hash: Hash,
+            selector: [u8; 4],
+            init_value: bool,
+        ) -> Option<AccountId> {
+            use ink::env::call::build_create;
+
+            let result = build_create::<
+                DefaultEnvironment,
+                constructors_return_value::ConstructorsReturnValueRef,
+            >()
+            .code_hash(code_hash)
+            .gas_limit(0)
+            .endowment(0)
+            .exec_input(ExecutionInput::new(Selector::new(selector)).push_arg(init_value))
+            .salt_bytes(&[0xDE, 0xAD, 0xBE, 0xEF])
+            .params()
+            .instantiate();
+
+            // NOTE: Right now we can't handle any `LangError` from `instantiate`, we can only tell
+            // that our contract reverted (i.e we see error from the Contracts pallet).
+            result.ok().map(|id| ink::ToAccountId::to_account_id(&id))
```

Comment:
How do you recommend handling instantiating other contracts inside a constructor?